services/QuillLMS/Gemfile

```diff
@@ -144,6 +144,7 @@ gem 'clever-ruby', '~> 2.0.1'
 
 # Memory profiling
 gem 'puma_worker_killer'
+gem 'memory_profiler', require: false
```

Comment:
Do we need this gem installed on prod? Maybe we should move to the `development` section?

Reply:
Good call.